llama-index-integrations/readers/llama-index-readers-document360/llama_index/readers/document360/entities/article.py

```diff
@@ -0,0 +1,135 @@
+class Article:
```

Comment:
Its probably best to make objects like these a pydantic model

Reply:
And then with that, no need for all the getters and setters

```
class Article(BaseModel):
  id: str
  title: str
  ...
```

If things can be `None`, then we should also use `Optional[str]` typing for example

And lastly, we can transform this into a dict easily with `.model_dump()` (useful for attaching as metadata to document objects)

Reply:
Updated

Reply:
All the fields in the entities are marked as Optional. This is because the actual API responses from Document360 sometimes do not match the expected schema mentioned in the API documentation.

(also mentioned it in the README file)